pkg/azuredisk/azure.go

```diff
@@ -20,26 +20,64 @@ import (
 	"fmt"
 	"os"
 
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog"
 	"k8s.io/legacy-cloud-providers/azure"
 )
 
 // GetCloudProvider get Azure Cloud Provider
-func GetCloudProvider() (*azure.Cloud, error) {
-	credFile, ok := os.LookupEnv("AZURE_CREDENTIAL_FILE")
-	if ok {
-		klog.V(2).Infof("AZURE_CREDENTIAL_FILE env var set as %v", credFile)
-	} else {
-		credFile = "/etc/kubernetes/azure.json"
-		klog.V(2).Infof("use default AZURE_CREDENTIAL_FILE env var: %v", credFile)
+func GetCloudProvider(kubeconfig string) (*azure.Cloud, error) {
+	klog.V(2).Infof("reading cloud config from secret")
```

Comment:
move `klog.V(2).Infof("reading cloud config from secret")` to L40

Reply:
done